examples/multiomics_mogonet/README.md

```diff
@@ -0,0 +1,60 @@
+# Multiomics Data Integration using Graph Convolutional Networks (MOGONET)
+
+## 1. Description
+
+Multimodal learning for multiomics data analysis is a promising research area in biomedical studies. The integration of multiple modalities (e.g., genomics, epigenomics, transcriptomics, proteomics, metabolomics , etc.) by capturing their complementary information provides a deeper understanding of most complex human diseases. This example is constructed by refactoring the [MOGONET](https://doi.org/10.1038/s41467-021-23774-w) [1], a multiomics integrative method for biomedical classification, within the PyTorch Geometric and PyTorch Lightning ecosystems.
```

Comment:
refactoring the MOGONET, : one noun is missing at the end, suggest --> refactoring the code of MOGONET 

"biomedical classification" is a weird phrase. Is the method for general biomedical data really?

within the PyTorch Geometric and PyTorch Lightning ecosystems --> using the PyTorch Geometric and PyTorch Lightning frameworks.